lib/bandit/ports.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Bandit.Ports do
+  @moduledoc """
+  Bandit helper functions and guards for dealing with port numbers.
+  """
```

Comment:
Internal libs like this use `@moduledoc false` to keep the docs focused on user-visible parts of Bandit.